lib/bolt/module_installer/puppetfile.rb

```diff
@@ -97,20 +97,33 @@ def assert_satisfies(specs)
           end
         end
 
-        return if unsatisfied_specs.empty?
-
+        versionless_mods = @modules.select { |mod| mod.is_a?(ForgeModule) && mod.version.nil? }
```

Comment:
@beechtom currently the test is failing because the module isn't being recognized as a `ForgeModule`. I tried with the ticket example `puppet/cron` and the pre-existing name `puppetlabs/yaml`